### DISCUSSION TITLE
Definicion del Proyecto Imposible

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-EM2022/Grupo7#59] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by JorEl057** [comment]
Chicos, que tal, como están :
Que pasó que no han avanzado en la realización del Proyecto Imposible? Tuvieron algún problema?

**Reply by chulkx** [reply]
Buenas tardes profe, como le va?
Mil disculpas por la demora en responder. Se me ha hecho difícil durante las ultimas semanas seguir el ritmo por cuestiones laborales, y coincidió con que todos los integrantes caímos en situaciones similares y ninguno pudo tomar la posta para hacer esta entrega.
Imagino que esto ya nos deja fuera de carrera para la regularización de la asignatura, pero de igual manera le consulto si la próxima instancia de evaluación la puedo realizar para seguir practicando (ya que desde la proxima semana voy a tener 2hs menos de trabajo por dia y podria intentarlo)  y poder rendir el final cuando corresponda.
